modules/swagger-codegen/src/main/resources/Java/libraries/jersey2/ApiClient.mustache

```diff
@@ -682,8 +682,9 @@ public class ApiClient {
 
     Entity<?> entity = serialize(body, formParams, contentType);
 
-    Response response;
+    Response response = null;
 
+    try {
     if ("GET".equals(method)) {
```

Comment:
@Moscagus Seems like the code within the try block is missing 2-space for indentation.

Please also run `./bin/java-all-petstore.sh` to update Java Petstore samples so that CIs can verify the change.